### DISCUSSION TITLE
New Designer

### DISCUSSION THREAD

**Discussion by timonmasberg** [primefaces/community#602] [PrimeNG] [answered]
Hi, on your [Visual Editor](https://designer.primeng.org/#/) it is stated there is a new Theme Designer (released in Q2 2023), but I can not find any new tool. Is it just not released yet? If so, what is the ETA?
Thanks in advance.

**Comment by cetincakiroglu** [comment, marked as answer]
Hello,

We are working tirelessly to bring you the Theme Designer. We've completed a portion of the work related to the Theme Designer, but there are still things we need to do, so this process will take a bit longer. We're aiming to release it, probably in the first quarter of 2024. The roadmap will be updated.

**Reply by dalenguyen** [reply]
Hi @cetincakiroglu, will the domain name for Theme Designer be the same? Or this is the new one? 

https://www.primefaces.org/designer-jsf/

**Reply by contis2908** [reply]
Error page

**Comment by armingdev** [comment]
What about upgrading older generated themes? Is there will be any way to update our current themes? I guess there will be changes in themes generated with the new designer tool.

**Reply by cetincakiroglu** [reply]
Hi,

We're preparing a "manual update guide" for the all templates. It will be available in primeng.org/guides section upcoming weeks.

**Comment by autonomobil** [comment]
Any update on this? Currently thinking about switching to primeng, but no theme editing (without a looooot of manual work) is a blocker

**Comment by B-o-s-s-e** [comment]
Any update on this? This is also a bit of a blocker for me. That new theme designer would make it a bit easier to get started.

**Comment by JimmyBarrientos** [comment]
Starting a new project soon, this would be beneficial. If I can get an actual roadmap I can push back on starting this project at my company and use this tool when it comes out. Are we looking at end of Q3, start of Q4 or end of Q4?

**Comment by dionisis84** [comment]
@cetincakiroglu I think that someone (or you) should answer the previous questions. We are all waiting for this tool and it would be good to have a roadmap. 🙂

**Comment by cagataycivici** [comment]
Hi guys, I'll be the dev of the new tool, currently working with the design team to finalize the design, in the meantime, did some POCs already.

https://primevue-image-theme.vercel.app/

Plan is to release is by the end of November, it will support PrimeVue 4, PrimeNG 18 and PrimeReact 11.

**Reply by TobiasHesselinkTopicus** [reply]
Hi, any update so far? Would love to use the new designer tool to create a new theme for PrimeNG. It's hard to create a new one now because of all the different variables for the components.

**Reply by hibace** [reply]
@cagataycivici hi, what is the status?

**Reply by matsandreassen-triona** [reply]
Anything?
